src/details/ArborX_DetailsMortonCode.hpp

```diff
@@ -45,6 +48,18 @@ unsigned int expandBitsBy2(unsigned int x)
   return x;
 }
 
+KOKKOS_INLINE_FUNCTION
+unsigned long long expandBitsBy2(unsigned long long x)
+{
+  x &= 0x1fffffllu;
+  x = (x | x << 32) & 0x1f00000000ffffllu;
+  x = (x | x << 16) & 0x1f0000ff0000ffllu;
+  x = (x | x << 8) & 0x100f00f00f00f00fllu;
+  x = (x | x << 4) & 0x10c30c30c30c30c3llu;
+  x = (x | x << 2) & 0x1249249249249249llu;
```

Comment:
In #633 you dropped the `u` suffix but now you are using it again.  You want to add it back to the functions that take `unsigned`?

Reply:
Yes, that was an oversight.